packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -0,0 +1,60 @@
+import { readdir } from "node:fs/promises";
+import path from "path";
+import vscode from "vscode";
+import { StartFileName } from "./const.js";
+import logger from "./log/logger.js";
+import { getDirectoryPath, normalizeSlashes } from "./path-utils.js";
+import { isFile } from "./utils.js";
+
+export const onStdioOut = (str: string) => {
+  str
+    .trim()
+    .split("\n")
+    .forEach((line) => logger.info(line));
+};
+export const onStdioError = (str: string) => {
```

Comment:
People can't figure out what these two methods for just by name which looks more like an event name... Why not just use the spawnExecutionAndLogToOutput which would log the stdio/stderr which is just what you want to do here, isn't it?

Reply:
`spawnExecutionAndLogToOutput` will log out all the logs in one string. I want to split it one line by one line which will be more clear.

Reply:
No, it will output in multiple lines too like this:
![image](https://github.com/user-attachments/assets/bccb895f-c1fa-4d5a-b1f5-4af49a9620b3)

Reply:
updated to use `spawnExecutionAndLogToOutput`